### DISCUSSION TITLE
How to show code that not allowed to change?

### DISCUSSION THREAD

**Discussion by hiimnhan** [cloudscape-design/components#806] [Q&A] [answered]
In this usage guideline https://cloudscape.design/components/code-editor/?tabId=usage, you guy said that we should not use Code Editor component to show code that not allow to change. So is there any component can i use ?

**Comment by taheramr** [comment, marked as answer]
Hello hiimnhan,

We don't currently have a readonly code component but it is in our plan, we'll keep you updated once we have something!

Till then you can use the native `pre` and `code` elements to display readonly code blocks in your application.
